design/spaces.go

```diff
@@ -48,6 +48,7 @@ var spaceRelationships = a.Type("SpaceRelationships", func() {
 	a.Attribute("workitems", relationGeneric, "Space can have one or many work items")
 	a.Attribute("codebases", relationGeneric, "Space can have one or many codebases")
 	a.Attribute("collaborators", relationGeneric, `Space can have one or many collaborators`)
+	a.Attribute("space-template", spaceTemplateRelation, `Space is constructed with a space template`)
```

Comment:
@kwk, may I suggest that you remove the `-` in `space-template` here ? I think it is error prone when we want to build a JSON document manually and it is somehow inconsistent with the other names that we use: `workitems`, `/spacetemplates`, etc. WDYT ?